src/compositions/__tests__/useConnector.js

```diff
@@ -0,0 +1,288 @@
+jest.unmock('instantsearch.js/es');
+import InstantSearch from '../../components/InstantSearch';
+import Index from '../../components/Index';
+import { createFakeClient } from '../../util/testutils/client';
+import { isVue3, isVue2, ref } from '../../util/vue-compat';
```

Comment:
This `vue-compat` needs to be replaced with `vue-compat/index-2-compositions.js` in case of vue 2. I don't know how to mock it that way. I had some issues with jest.mock().

For now, we're testing only on vue 3.

Reply:
could it be required inside the conditional, or a file for vue 3 and a file for vue 2?

Reply:
this `vue-compat` is also imported within the source `useConnector.js`